### PR TITLE
feat: Allow disconnecting agent bots

### LINKED ISSUE #6242: Issues with bot UI screens
I am not sure if these are bugs or as intended?

Issue 1 - Once you've configured the bot, you can't delete it in super_admin
Issue 2 - If you assign a bot to an org, and assign the bot to an inbox, you can't "unassign" it.

Issue 2 screenshot
image

This is a self hosted, up to date version of Chatwoot.

### PR DESCRIPTION
**Pull request by fayazara**
# Pull Request Template

## Description

This PR allows you to disconnect agent bots from inboxes.
https://www.loom.com/share/6f09b7085b4b406780c99c2a7595ff51

Fixes #6242 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Tested locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/inbox/components/BotConfiguration.vue [hunk 1]
```diff
@@ -86,6 +99,26 @@ export default {
...
+    async disconnectBot() {
+      try {
+        await this.$store.dispatch('agentBots/disconnectBot', {
+          inboxId: this.inbox.id,
+        });
+        this.showAlert(
+          this.$t('AGENT_BOTS.BOT_CONFIGURATION.DISCONNECTED_SUCCESS_MESSAGE')
+        );
+      } catch (error) {
+        this.showAlert(
```
Comment (muhsin-k) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/components/BotConfiguration.vue:111]: Check if there any error message exists in the error payload.

**Review by scmmishra — APPROVED** [review]
LGTM, just need to resolve the comment from @muhsin-k

**Review by iamsivin — APPROVED** [review]
LGTM

File: app/javascript/dashboard/routes/dashboard/settings/inbox/components/BotConfiguration.vue (hunk 1 above)
Comment (fayazara) [reply app/javascript/dashboard/routes/dashboard/settings/inbox/components/BotConfiguration.vue:111]: @muhsin-k Handled this, please have a look

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**